assignments/prototypes.js

```diff
@@ -83,28 +150,114 @@
       height: 4,
     },
     healthPoints: 10,
-    name: 'Lilith',
-    team: 'Forest Kingdom',
+    name: 'Rowen Hashiba',
+    team: 'Ronin Warriors',
     weapons: [
       'Bow',
       'Dagger',
     ],
-    language: 'Elvish',
+    language: 'Ancient Japanese',
   });
 
   console.log(mage.createdAt); // Today's date
```

Comment:
Just for added fun you can look into how to parse date formats as well so that you're not only getting back whatever javascript decides to send you.